rt_utils/image_helper.py

```diff
@@ -115,41 +130,68 @@ def validate_contours(contours: list):
     if len(contours) == 0:
         raise Exception("Unable to find contour in non empty mask, please check your mask formatting")
 
+def get_pixel_to_patient_transformation_matrix(series_data):
+    """
+    https://nipy.org/nibabel/dicom/dicom_orientation.html
+    """
+
+    first_slice = series_data[0]
+
+    offset = first_slice.ImagePositionPatient
+    row_spacing, column_spacing = first_slice.PixelSpacing
+    slice_spacing = get_spacing_between_slices(series_data)
+    row_direction, column_direction, slice_direction = get_slice_directions(first_slice)
+
+    mat = np.identity(4, dtype=np.float32)
+    mat[:3, 0] = row_direction * row_spacing
+    mat[:3, 1] = column_direction * column_spacing
+    mat[:3, 2] = slice_direction * slice_spacing
+    mat[:3, 3] = offset
+
+    return mat
+
+def get_patient_to_pixel_transformation_matrix(series_data):
+    return np.linalg.inv(get_pixel_to_patient_transformation_matrix(series_data))
```

Comment:
I've never had a chance to work with matrices in practice. Since these are just linear transformations we won't have to worry about precision errors or un-invertible matrices right?

Reply:
Yup, these are affine transformations, so they will always have an inverse.

But this line was just me being lazy. Removed the call to inv in the next commit and calculated the inverse by hand, so we don't have to worry about precision errors.